shoot.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "rspec", "~> 2"
 
   spec.add_dependency "selenium-webdriver", "~> 2"
-  spec.add_dependency "capybara", "~> 2"
+  spec.add_dependency "capybara", "2.4.4"
```

Comment:
to reflect that the gem does not support versions above it.
(not related to my change, actually)